src/js/profile/wearable/widget/wearable/Scrollview.js

```diff
@@ -162,11 +163,13 @@
 				scroller.appendChild(fragment);
 
 				if (ns.support.shape.circle) {
-					if (scroller) {
+					enableCircularScrollbar = ns.getConfig("enableCircularScrollbar");
+
+					if (scroller && enableCircularScrollbar) {
```

Comment:
What does enableCircularScrollbar mean? Do you mean wearable scroll?

Reply:
It means turning the circular scollbar for wearable on and off.

Reply:
got it.